daisy_workflows/image_build/enterprise_linux/kickstart/el8-post.cfg

```diff
@@ -37,7 +37,7 @@ rpmkeys --import /etc/pki/rpm-gpg/google-rpm-package-key.gpg
 rpmkeys --import /etc/pki/rpm-gpg/google-key.gpg
 
 # Install GCE guest packages.
-dnf install -y __PACKAGES__ python27
+dnf install -y google-compute-engine google-osconfig-agent google-cloud-sdk gce-disk-expand python27 python36 __PACKAGES__
```

Comment:
We need to use python2 (instead of python27) here. See: https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1016/files